### PR TITLE
WIP: MGMT-7336: add labels to the api

### PR DESCRIPTION
**Pull request by pkliczewski**
Provides api to update host labels.

Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
This PR depends on https://github.com/openshift/assisted-service/pull/2320

File: discovery-infra/test_infra/assisted_service_api.py [hunk 1]
```diff
@@ -253,6 +253,11 @@ def set_pull_secret(self, cluster_id: str, pull_secret: str) -> models.cluster.C
         update_params = models.ClusterUpdateParams(pull_secret=pull_secret)
         return self.update_cluster(cluster_id=cluster_id, update_params=update_params)
 
+    def set_labels(self, cluster_id: str, hosts_with_labels) -> models.cluster.Cluster:
+        log.info("Setting labels for hosts %s in cluster %s", hosts_with_labels, cluster_id)
+        update_params = models.ClusterUpdateParams(host_labels=hosts_with_labels)
+        return self.update_cluster(cluster_id=cluster_id, update_params=update_params)
```
Comment (lalon4) [inline discovery-infra/test_infra/assisted_service_api.py:259]: should be v2_update_cluster

Comment (pkliczewski) [reply discovery-infra/test_infra/assisted_service_api.py:259]: This method is called on line 263. Here we call local method.

**Comment by eliorerz** [comment]
@pkliczewski Any update on this one?

**Comment by pkliczewski** [comment]
@eliorerz this work will be continued by @rewantsoni

**Pull request closed without merging**